Automattic/wp-calypso
# Enable VideoPress Privacy Setting on wpcom simple sites.

#### Changes proposed in this Pull Request

* Removes the hard-coded false and thus enables VideoPress Privacy setting on all wpcom simple sites.

#### Testing instructions



* open your site's media library: http://calypso.localhost:3000/media/yoursite.wordpress.com
* Upload or select a VideoPress video.
* Edit. You see privacy settings select box.
* Changing values persists and the privacy setting is applied.



Related to #
